src/apps/search-header/search-header.tsx

```diff
@@ -349,8 +381,42 @@ const SearchHeader: React.FC = () => {
           autosuggestCategoryList={autosuggestCategoryList}
           isLoading={isLoading}
         />
+        {isHeaderDropdownOpen && (
+          <div
+            className="header__menu-dropdown"
+            data-cy="search-header-dropdown"
+          >
+            <ul>
+              <li>
+                <button
+                  ref={headerDropdownRef}
+                  type="button"
+                  role="menuitem"
+                  className="header__menu-dropdown-item cursor-pointer"
+                  onClick={() => redirectTo(advancedSearchUrl)}
+                  onKeyUp={(e) => {
+                    if (e.key === "Enter") {
+                      return redirectTo(advancedSearchUrl);
+                    }
+                    if (
+                      e.key === "ArrowDown" ||
+                      e.key === "ArrowUp" ||
+                      e.key === "Escape"
+                    ) {
+                      return setIsHeaderDropdownOpen(false);
+                    }
+                    return null;
+                  }}
+                  onBlur={() => setIsHeaderDropdownOpen(false)}
+                >
+                  {t("headerDropdownItemAdvancedSearchText")}
+                </button>
+              </li>
+            </ul>
+          </div>
+        )}
```

Comment:
This is a rather big block of a new feature. Could we throw that into a separate component?